wandb/integration/keras/callbacks/metrics_logger.py

```diff
@@ -0,0 +1,79 @@
+import sys
+from typing import Any, Dict, Optional, Union
+
+from tensorflow.keras import callbacks  # type: ignore
+
+import wandb
+from wandb.integration.keras.keras import patch_tf_keras
+from wandb.sdk.lib import telemetry
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+
+LOG_STRATEGY = Literal["epoch", "batch"]
+
+
+patch_tf_keras()
+
+
+class WandbMetricsLogger(callbacks.Callback):
```

Comment:
@ayulockin: there are a couple (legacy) places where we're using "WandB" (should fix), but generally prefer "Wandb".